hoomd/md/constrain.py

```diff
@@ -331,14 +331,17 @@ def __init__(self):
         self._add_typeparam(body)
         self.body.default = None
 
-    def create_bodies(self, state, charges=None):
+    def create_bodies(self, state, charges=None, masses=None):
         r"""Create rigid bodies from central particles in state.
 
         Args:
             state (hoomd.State): The state in which to create rigid bodies.
             charges (dict[str, list[float]]): (optional) The charges for each of
                 the constituent particles, defaults to ``None``. If ``None``,
                 all charges are zero. The keys should be the central particles.
+            masses (dict[str, list[float]]): (optional) The masses for each of
+                the constituent particles, defaults to ``None``. If ``None``,
+                all masses are one. The keys should be the central particles.
```

Comment:
Just want to check: is the current behavior of this method to assign mass 1 to all constituents (maybe by default because it goes through the snapshot)? If yes, then this is good as is. If no, we should consider if it should be zero.